testing/impeller_golden_tests_output.txt

```diff
@@ -185,6 +185,9 @@ impeller_Play_AiksTest_BlurHasNoEdge_Vulkan.png
 impeller_Play_AiksTest_BlurredRectangleWithShader_Metal.png
 impeller_Play_AiksTest_BlurredRectangleWithShader_OpenGLES.png
 impeller_Play_AiksTest_BlurredRectangleWithShader_Vulkan.png
+impeller_Play_AiksTest_CanConvertTriangleFanToTriangles_Metal.png
```

Comment:
Ah, these are being moved from non-golden tests to golden tests?  Excellent.

Reply:
Yup!